daprdocs/content/en/operations/components/setup-pubsub/setup-pubsub-overview.md

```diff
@@ -30,11 +32,17 @@ spec:
 ...
 ```
 
-The type of pub/sub is determined by the `type` field, and things like connection strings and other metadata are put in the `.metadata` section.
+The type of pub/sub is determined by the `type` field, and properties such as connection strings and other metadata are put in the `.metadata` section.
 Even though you can put plain text secrets in there, it is recommended you use a [secret store]({{< ref component-secrets.md >}}) using a `secretKeyRef`
 
-Visit [this guide]({{< ref "howto-publish-subscribe.md#step-2-publish-a-topic" >}}) for instructions on configuring pub/sub components.
+Visit [this HowTo guide]({{< ref "howto-publish-subscribe.md#step-2-publish-a-topic" >}}) for instructions on configuring and using pub/sub components.
```

Comment:
"How To" or "How-To", probably not "HowTo". In this case "this guide" is probably better

Reply:
Fixed to "this guide"